.gitlab-ci.yml

```diff
@@ -266,6 +267,27 @@ E2E Test (upload to s8s):
     - export DRY_RUN=${DRY_RUN:-0} # default to 0 if not specified
     - make e2e-build-upload ARTIFACTS_PATH="artifacts/e2e"
 
+# ┌────────────────────────────┐
+# │ Benchmark Test app upload: │
+# └────────────────────────────┘
+
+Benchmark Test (upload to s8s):
+  stage: benchmark-test
+  rules:
+    - if: '$CI_COMMIT_BRANCH == $DEVELOP_BRANCH'
+  variables:
+    XCODE: "15.3.0"
+    OS: "17.4"
+  artifacts:
+    paths:
+      - artifacts
+    expire_in: 2 weeks
+  script:
+    - ./tools/runner-setup.sh --xcode "$XCODE" --iOS --os "$OS" --datadog-ci # temporary, waiting for AMI
```

Comment:
Things have changed on `develop` recently (through #1975). Because we target `macos:sonoma` [by default](https://github.com/DataDog/dd-sdk-ios/blob/088d1918faac5f89c2e8b7c801d17f5a7c57b315/.gitlab-ci.yml#L27-L30), I'd suggest using default Xcode (15.4) which comes with the latest iOS runtime (17.5). Effectively, we only need to ensure that `$DEFAULT_XCODE` is the active Xcode. 

💡 Also, there's no need now for `--iOS` and `--os "17.5"` as both come pre-installed in the runner's image.

```suggestion
  artifacts:
    paths:
      - artifacts
    expire_in: 2 weeks
  script:
    - ./tools/runner-setup.sh --xcode "$DEFAULT_XCODE" --datadog-ci
```

Reply:
@maxep ☝️ we may need that one too 🙂